### ISSUE TITLE
A11y: Connect Radio buttons with the control elements

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#24930]
The labels of the control fields are not linked to the associated control
elements (example marked in blue). Therefore, when the control fields are
focused, the caption is not read out by the screen reader, which means that
blind users do not experience the functionality of the selection fields.

This problem applies to all control fields in Settings.

Suggestion: The labels should be surrounded by a label tag, which is linked to the associated control element using the for attribute and an id.

**Comment by t3chguy** [comment]
@daniellekirkwood your issue says radio buttons but your screenshot highlights checkboxes, could you please clarify?

**Comment by t3chguy** [comment]
The labels are linked using `for` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label - 
![image](https://user-images.githubusercontent.com/2403652/232026782-201f090b-6f18-495b-aca7-26776c7d7c01.png)

**Comment by t3chguy** [comment]
https://github.com/vector-im/element-web/assets/2403652/cb794aba-6559-4d79-8f27-763dcc28bbed

Closing as cannot reproduce
